### DISCUSSION TITLE
spam

### DISCUSSION THREAD

**Discussion by vinothmp21102005** [community/community#84047] [New to GitHub]
idk why am i marked as spam can anyone help

**Comment by queenofcorgis** [comment]
Hi @vinothmp21102005 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by crrusucosmin** [comment]
Hi! My GitHub account rusucosmin got disabled after I fell into the trap of this spam message - I need back access to the account, or I would like to get all my private repositories so I can move then somewhere else

**Reply by queenofcorgis** [reply]
Hi - please refer to my message above - our Support team are the only ones who are able to assist in account-specific situations.
